### PR TITLE
Gen 3-4: Always break opponent screens with brick break

### PR DESCRIPTION
**Pull request by pacmanboss256**

### CODE REVIEW

File: data/mods/gen3/moves.ts [hunk 1]
```diff
@@ -98,6 +98,9 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		accuracy: 100,
 		ignoreAccuracy: true,
 	},
+	brickbreak: {
+		inherit: true,
```
Comment (Karthik99999) [inline data/mods/gen3/moves.ts:102]: you don't need to define inherit here, gen 3 already inherits from gen 4

File: test/sim/moves/brickbreak.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ describe('Brick Break', function () {
 		assert(battle.p2.sideConditions['reflect']);
 
 		battle.makeChoices('move brickbreak', 'move splash');
+		common.saveReplay(battle, 'brickbreak.js');
```
Comment (Karthik99999) [inline test/sim/moves/brickbreak.js:52]: not needed

Comment (pacmanboss256) [reply test/sim/moves/brickbreak.js:52]: whoops that was for me to test

**Comment by pacmanboss256** [comment]
this worked on my end so idk whats up, but i have to redo this to make sure the move hits

**Comment by pacmanboss256** [comment]
ignore the hardcoding

**Pull request closed without merging**